packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -268,6 +268,24 @@ enum StackFit {
   passthrough,
 }
 
+@Deprecated(
```

Comment:
this deprecation should also go below the doc comment.

Reply:
Done.